### ISSUE TITLE
Hardware License

### ISSUE THREAD

**Issue by dbrgn** [gcormier/megadesk#36]
Hi, I wondered about the license of this project and only found the GPL.

While the GPL works great for the firmware, it doesn't really work with hardware. Maybe it would be worth considering putting the hardware unter an OSHW license, e.g. the [TAPR Open Hardware License](https://tapr.org/the-tapr-open-hardware-license/) or the [CERN Open Hardware License](https://en.wikipedia.org/wiki/CERN_Open_Hardware_Licence).

**Comment by gcormier** [comment]
I'm good with leaving it GPL based on this information


> I spent quite a bit of time looking into this, and there is not really a useful solution. For some general background try https://www.publicknowledge.org/news-blog/blogs/open-source-hardware-and-law1
> 
> In a nutshell, GPL (and all other software licenses) rely on software being something that can be subject to copyright. In general hardware can not be copyrighted, because copyright is only granted to creative or artistic works, but with some weird exceptions like software, IC masks, yacht designs (!). "Useful articles or utility works" are not generally subject to copyright, but some powerful industrial lobbies got some concessions, as otherwise a software "work" would not be protected under IP laws, although specific software algorithms can be patented.
> 
> So anyhoo, software is an oddity in copyright laws, but "copyleft" licenses can turn copyright law to its advantage. There is no such ability with hardware. The only way to protect a hardware design is to embody a patentable idea that the design relies on. "Data files" for engineering use are not software, and are not "creative" (in terms of the law), so it is unlikely they have any form of protection. That doesn't stop many companies attempting to claim copyright on data, the threat of legal action even if unfounded in law is often enough to scare people off.
> 
> Legally, you can use copyright to protect your drawings, but you can't protect the actual circuit, nor the PCB layout, and therefore you can't insist on share-alike clauses to be followed. What we are left with is a "social contract", i.e. a statement that a design is shared for general use but please be nice and share your modifications.
> 
> There are some OSHW specific licenses, such as CERN, OSHWA, OHANDA, these have the same drawback as GPL, copyright gives little protection to hardware designs. But they can be a nice way to stamp your project as being in the OSHW spirit.
> 
> tldr; use GPL or LGPL, CC-BY-SA, MIT, etc as you like, as a statement of intent, but realise they have little legal teeth. Other OSHW oriented licenses are equally ineffective to protect or control the use of electronic or hardware designs.
